### DISCUSSION TITLE
How to inline SVG?

### DISCUSSION THREAD

**Discussion by bytrangle** [withastro/docs#2473] [Ideas]
I suppose it's not possible to inline SVG in Astro without installing additional packages? According to this [link](https://docs.astro.build/en/guides/imports/), you can import SVG, but it needs to be wrapped in `<img>` tag.

**Comment by lj-n** [comment]
You can use SVG just like HTML in Astro components:
```astro
---
const color = "green";
---

<svg width="100" height="100">
  <circle cx="50" cy="50" r="40" fill={color} />
</svg>
```

**Comment by ArmandPhilippot** [comment]
We're closing stale and outdated discussion to keep only the discussions still relevant to the current project version and/or the current docs state.

It seems the question as been resolved, so I'm closing this. By the way, since Astro 5.7 you can also use [SVG components](https://docs.astro.build/en/guides/images/#svg-components) now!

If you think this is a mistake and that this discussion should remain open, please feel free to ping me, I'll take a second look at it.

Thank your for your contribution!
